src/clr-angular/forms/select/select-menu.html

```diff
@@ -0,0 +1,8 @@
+<!--
+* Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+* This software is released under MIT license.
+* The full license information can be found in LICENSE in the root directory of this project.
+-->
+
+<!--Option Groups and Options will be projected here-->
+<ng-content></ng-content>
```

Comment:
Are we expecting this template to be expanded in the future? Otherwise, can we have it in as `TemplateUrl:`?